docker-compose.yml

```diff
@@ -1101,6 +1152,7 @@ volumes:
   mosca-redis-volume:
   influxdb-volume:
   acl-redis-volume:
+  http-agent-cert-side-volume:
```

Comment:
Just an aesthetic comment... I think the volume name could follow the "dojot pattern", so for this you need to change `http-agent-cert-side-volume` to `http-agent-cert-sidecar-volume`.